model-optimizer/extensions/ops/mvn.py

```diff
@@ -49,18 +50,28 @@ def backend_attrs(self):
     def infer(node: None):
         input_shape = node.in_node(0).shape
         name = node.soft_get('name', node.id)
-        axes = node.axes
-        if axes is not None:
-            if 0 in axes:
-                raise Error('Reduction over the batch dimension in node "{}" '
-                            'is not supported by the backend.'.format(name))
-            for i in range(2, len(input_shape)):
-                if i not in axes:
-                    raise Error(
-                        'Reduction over spatial dimensions in node "{}" '
-                        'is obligatory for the backend.'.format(name))
-            if 1 in axes and not node.across_channels:
-                raise Error('Inconsistent values of axes ({}) and across_channels ({}) parameters '
-                            'in node "{}".'.format(str(axes), str(node.across_channels), name))
+
+        if node.axes is not None and node.across_channels is not None:
+            raise Error('Either axes or across_channels can be set for the MVN in node "{}".'.format(name))
+
+        if node.across_channels is None:
+            if node.axes is not None:
+                # normalizing (replacing -1 with actual index)
+                axes_data_value = node.axes
+                axes = [axes_data_value.item()] if axes_data_value.size == 1 else axes_data_value
+                axes = [get_canonical_axis_index(input_shape, a) for a in axes]
+                # deduce across_channels from the axes, e.g. if the first axis is included (assuming batch is zero axis)
+                node.across_channels = int(1 in axes)
```

Comment:
why 1? is it layout dependent?

Reply:
Yeah. Good point. We should get identify the feature dim like this:
`feature_dim = get_features_dim(node.graph.graph['layout'], len(input_shape))`

and then check: `node.across_channels = int(feature_dim in axes)`

Reply:
get_features_dim supports only 4 and 5d shapes

Reply:
Ok. Then add a check that if `4 <= len(input_shape) <= 5` then use `get_features_dim` else check that 1 is in axes as before